src/model/annotation.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from datetime import datetime
+from enum import Enum, auto
+
+
+class AnnotationType(Enum):
+    REVIEW = auto()
+    OTHER = auto()
+
+
+class Annotation:
+    spdx_id: str
+    annotation_type: AnnotationType
+    annotator: str
+    annotation_date: datetime
+    comment: str
+
+    def __init__(self, spdx_id: str, annotation_type: AnnotationType, annotator: str, annotation_date: datetime,
+                 comment: str):
+        self.spdx_id = spdx_id
+        self.annotation_type = annotation_type
+        self.annotator = annotator
+        self.annotation_date = annotation_date
+        self.comment = comment
```

Comment:
Hm... true, now it's inconsistent. Just like the [JSON schema](https://github.com/spdx/spdx-spec/blob/v2.3/schemas/spdx-schema.json#L30) and [RDF schema](https://spdx.org/rdf/ontology/spdx-2-3/#d4e2001).

I agree with renaming it, since it's the main content of the annotation, as you pointed out.